Adjust some default settings (default shipped .conf files) for 
openSUSE and openSUSE based appliances.

Removing default LiveCD/LiveUSB user is not mandatory, 
but "linux" user is no longer needed after installation.
Note, that if user not exist, then Camamares crash!
Thus here we justion mention this option, but not enabling.

diff --git a/settings.conf b/settings.conf
--- a/settings.conf
+++ b/settings.conf
@@ -141,17 +141,19 @@
   - fstab
 #  - plymouthcfg
 #  - zfshostid
-  - initcpiocfg
-  - initcpio
+#  - initcpiocfg
+#  - initcpio
+#  - removeuser
   - users
   - displaymanager
   - networkcfg
   - hwclock
-  - services-systemd
-#  - dracut
-  - initramfs
-#  - grubcfg
+#  - services-systemd
+  - dracut
+#  - initramfs
+  - grubcfg
   - bootloader
+  - packages
   - umount
 - show:
   - finished
